server.js

```diff
@@ -0,0 +1,31 @@
+//Install express server
+const express = require("express");
+const path = require("path");
+const cookieSession = require("cookie-session");
+const cookieParser = require("cookie-parser");
+
+const app = express();
+
+app.use(cookieParser());
+
+
+app.use(
+  
+  cookieSession({
```

Comment:
what are we storing in a cookie?

Reply:
Please we are storing the environment variables that we'll need in the service and the session secret key is to protect them